src/Path/Pretty.hs

```diff
@@ -72,10 +86,17 @@ prettyMeta :: Pretty a => a -> Doc
 prettyMeta n = dullblack (bold (pretty '?' <> pretty n))
 
 
+prettySpan :: Span -> Doc
```

Comment:
The `Pretty` instance for `Span` wasn’t a great idea, since it isn’t intended to be used on its own, so we’ve written it as a function & moved it here.